### PR TITLE
Uf 5059 add api resource for reading events connected to an errand

### PR DESCRIPTION
**Pull request by josc04**

### CODE REVIEW

File: src/main/java/se/sundsvall/supportmanagement/api/EventResource.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		private static final long serialVersionUID = 1L;
+
+		public PagedEvent() {
+			super(Collections.emptyList());
+		}
+	}
+
+	@GetMapping(path = "/errands/{id}/events", produces = { APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE })
+	@Operation(summary = "Read errand events", description = "Returns all existing events for the errand that matches the provided id")
+	@ApiResponse(responseCode = "200", description = "Successful operation", content = @Content(mediaType = APPLICATION_JSON_VALUE, schema = @Schema(implementation = PagedEvent.class)))
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/supportmanagement/api/EventResource.java:53]: Why not:
`@ApiResponse(responseCode = "200", description = "Successful operation", content = @Content(mediaType = APPLICATION_JSON_VALUE), useReturnTypeSchema = true)`

Then remove: PagedEvent 

Have not tested, but will probably wortk.

Comment (josc04) [reply src/main/java/se/sundsvall/supportmanagement/api/EventResource.java:53]: I tried that but it didn't work - which was the reason that I added the PagedEvent class. 

With suggested solution:
image

With PagedEvent:
image

Comment (foobar-1337) [reply src/main/java/se/sundsvall/supportmanagement/api/EventResource.java:53]: It works if you do it like this:
`@ApiResponse(responseCode = "200", description = "Successful operation", useReturnTypeSchema = true)`

Comment (foobar-1337) [reply src/main/java/se/sundsvall/supportmanagement/api/EventResource.java:53]: It seems to work if you do it like this:
`@ApiResponse(responseCode = "200", description = "Successful operation", useReturnTypeSchema = true)`

Comment (josc04) [reply src/main/java/se/sundsvall/supportmanagement/api/EventResource.java:53]: Good finding 👍, I'll refactor to use returnTypeSchema instead.

**Review by foobar-1337 — APPROVED** [review]

**Pull request merged**